plotly/tests/test_core/test_stream/test_stream.py

```diff
@@ -10,7 +10,7 @@
 import time
 
 from nose.plugins.attrib import attr
-from nose.tools import raises
+from nose.tools import raises, eq_
```

Comment:
😿 we should really switch this over. Mind if I make a PR into yrs to update this? We should be doing class-based testing using a `TestCase`